_src/static/sass/utilities/_variables.scss

```diff
@@ -25,6 +25,8 @@ $baseline-horizontal:    $baseline;
 // color: config
 $transparent:       rgba(0, 0, 0, 0);
 
+$white:             rgb(255, 255, 255);
+$black:             rgb(0, 0, 0);
 $gray:              rgb(127, 127, 127);
```

Comment:
Hmm, looks like this is already defined in https://github.com/edx/ux-pattern-library/blob/clrux/base-theme-settings/_src%2Fstatic%2Fsass%2Futilities%2F_variables.scss#L92. If the Sass Map makes it hard to customize this value, I'd recommend using the ``palette``function to define a ``$gray`` var. Perhaps something like:

```scss
$gray: palette(grayscale, base);
```